workspaces/spectacle/src/helpers.ts

```diff
@@ -85,84 +85,70 @@ export function buildEndpointChanges(
   shapeQueries: shapes.GraphQueries,
   sinceBatchCommitId?: string
 ): EndpointChanges {
-  // Sorted in reverse order - the latest change takes precedence
-  const sortedBatchCommits = endpointQueries
-    .listNodesByType(endpoints.NodeType.BatchCommit)
-    .results.sort((a: any, b: any) => {
-      return a.result.data.createdAt < b.result.data.createdAt ? 1 : -1;
-    }) as endpoints.BatchCommitNodeWrapper[];
-
-  let deltaBatchCommits: endpoints.BatchCommitNodeWrapper[];
-
-  if (sinceBatchCommitId) {
-    const sinceBatchCommit: any = endpointQueries.findNodeById(
-      sinceBatchCommitId!
-    );
-    deltaBatchCommits = sortedBatchCommits.filter(
-      (batchCommit: any) =>
-        batchCommit.result.data.createdAt >
-        sinceBatchCommit!.result.data.createdAt
-    );
-  } else {
-    deltaBatchCommits = sortedBatchCommits;
-  }
+  const deltaBatchCommits = getDeltaBatchCommitsForEndpoints(
+    endpointQueries,
+    // In this case specifically, we want _all_ endpoint changes
+    sinceBatchCommitId || ALL_BATCH_COMMITS
```

Comment:
This function is confusing - we want to query _all_ batch commits sometimes (in the changelog view). I want to reconcile this, but I don't know if I'll have time to get around to this. I'm adding this constant so that it's _explicit_ that we want to query all changes in all batches (which results in everything being marked as added)